diff --git a/CMake/ChimaeraCommonConfig.cmake b/CMake/ChimaeraCommonConfig.cmake
index 79741d4..a181046 100644
--- a/CMake/ChimaeraCommonConfig.cmake
+++ b/CMake/ChimaeraCommonConfig.cmake
@@ -59,7 +59,7 @@ endif()
 
 # Pybind11
 if(CHIMAERA_ENABLE_PYTHON)
-    find_first_path_python()
+#    find_first_path_python()
     find_package(pybind11 REQUIRED)
     set(OPTIONAL_LIBS pybind11::embed)
 endif()
