diff --git a/hrun/src/CMakeLists.txt b/hrun/src/CMakeLists.txt
index 5c9ee288..b1e650bb 100644
--- a/hrun/src/CMakeLists.txt
+++ b/hrun/src/CMakeLists.txt
@@ -15,7 +15,7 @@ target_link_libraries(hrun_client
   yaml-cpp
   cereal::cereal
   thallium
-  -ldl -lrt -lc -pthread)
+  -ldl -lrt -lc -pthread OpenMP::OpenMP_CXX)
 
 #------------------------------------------------------------------------------
 # Build Hrun Runtime Library
@@ -109,4 +109,4 @@ if(HERMES_ENABLE_COVERAGE)
   set_coverage_flags(hrun_runtime)
   set_coverage_flags(hrun_start_runtime)
   set_coverage_flags(hrun_stop_runtime)
-endif()
\ No newline at end of file
+endif()
