--- a/benchmark/allocator/allocator.cc
+++ b/benchmark/allocator/allocator.cc
@@ -14,5 +14,6 @@
 #include "test_init.h"
 #include "omp.h"
 
+#include <random>
 #include <string>
 #include "hermes_shm/data_structures/ipc/string.h"
--- a/test/unit/allocators_mpi/allocator_mpi.cc
+++ b/test/unit/allocators_mpi/allocator_mpi.cc
@@ -14,6 +14,8 @@
 #include "test_init.h"
 #include "basic_test.h"
 
+#include <random>
+
 struct Record {
   char *data;
   size_t size;
