--- CMakeLists.txt	2024-08-15 16:15:33.000000000 -0500
+++ CMakeLists.txt	2025-01-08 22:56:43.064916100 -0600
@@ -1,6 +1,5 @@
 cmake_minimum_required(VERSION 3.10)
 project(hermes)
-
 #-----------------------------------------------------------------------------
 # Define Constants
 #-----------------------------------------------------------------------------
@@ -181,8 +180,7 @@
 if(HERMES_ENABLE_VFD)
     set(HERMES_REQUIRED_HDF5_VERSION 1.14.0)
     set(HERMES_REQUIRED_HDF5_COMPONENTS C)
-    find_package(HDF5 ${HERMES_REQUIRED_HDF5_VERSION} CONFIG NAMES hdf5
-            COMPONENTS ${HERMES_REQUIRED_HDF5_COMPONENTS} shared)
+    find_package(HDF5 REQUIRED)
     if(HDF5_FOUND)
         message(STATUS "found HDF5 ${HDF5_VERSION} at ${HDF5_INCLUDE_DIR}")
         set(HDF5_HERMES_VFD_EXT_INCLUDE_DEPENDENCIES
@@ -190,7 +188,8 @@
                 ${HDF5_INCLUDE_DIR})
         set(HDF5_HERMES_VFD_EXT_LIB_DEPENDENCIES
                 ${HDF5_HERMES_VFD_EXT_LIB_DEPENDENCIES}
-                ${HDF5_C_SHARED_LIBRARY})
+                ${HDF5_C_LIBRARIES})
+        message(STATUS "found HDF5 ${HDF5_C_LIBRARIES}")		
     else()
         # Allow for HDF5 autotools builds
         find_package(HDF5 ${HERMES_REQUIRED_HDF5_VERSION} MODULE REQUIRED
@@ -325,4 +324,4 @@
         ${CMAKE_INSTALL_PREFIX}/cmake
 )
 
-install(DIRECTORY include DESTINATION ${CMAKE_INSTALL_PREFIX})
\ No newline at end of file
+install(DIRECTORY include DESTINATION ${CMAKE_INSTALL_PREFIX})
